R/use_lintr.R

```diff
@@ -43,5 +43,37 @@ use_lintr <- function(path = ".", type = c("tidyverse", "full")) {
     )
   )
   write.dcf(the_config, config_file, width = Inf)
+
+  if (file.exists(file.path(path, "DESCRIPTION"))) {
+    # Some OS can only normalize a path if the associated file or folder exists, so the path needs to be re-normalized
+    tryCatch({
+      pkg_path <- normalizePath(path, mustWork = TRUE, winslash = "/")
+      config_file <- normalizePath(file.path(path, lintr_option("linter_file")), mustWork = TRUE, winslash = "/")
+    }, error = function(e) {
+      stop("No entry could be added to the .Rbuildignore.", call. = FALSE)
```

Comment:
This sounds like a warning is in order. The main side effect was caused at this point anyway.